src/hooks/useLocale.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState, useEffect } from "react";
+import Spanish from "../lang/es-US.json";
+import English from "../lang/en-US.json";
```

Comment:
⛏️ Rename `Spanish` to `es-US` and `English` to `en-US`.

Reply:
It's not liking the dashes.  It works fine if I leave them out.  Any advice?
![Screenshot 2021-09-13 132953](https://user-images.githubusercontent.com/73480835/133137576-a2c9e86a-3c14-49b3-b4f8-0078a8414a6e.png)

Reply:
What about something like this instead?

```js
    case 'es-US':
      return import('../translations/es-US.json');
    default:
      return import('../translations/en-US.json');
  }
```

Reply:
It doesn't seem to like that either, unfortunately
![Screenshot 2021-09-13 163301](https://user-images.githubusercontent.com/73480835/133159473-b29495f3-aec9-43cb-b850-9595cf15d955.png)

Reply:
Is it just a linting issue? Does it work?

Reply:
No - it's not working at all.  Not just a linting issue.